jquery.color.js

```diff
@@ -147,26 +151,27 @@ each( spaces, function( spaceName, space ) {
 		type: "percent",
 		def: 1
 	};
-});
+} );
 
 function clamp( value, prop, allowEmpty ) {
 	var type = propTypes[ prop.type ] || {};
 
 	if ( value == null ) {
-		return (allowEmpty || !prop.def) ? null : prop.def;
+		return ( allowEmpty || !prop.def ) ? null : prop.def;
 	}
 
 	// ~~ is an short way of doing floor for positive numbers
 	value = type.floor ? ~~value : parseFloat( value );
 
 	if ( type.mod ) {
+
 		// we add mod before modding to make sure that negatives values
 		// get converted properly: -10 -> 350
-		return (value + type.mod) % type.mod;
+		return ( value + type.mod ) % type.mod;
 	}
 
 	// for now all property types without mod have min and max
-	return 0 > value ? 0 : type.max < value ? type.max : value;
+    return Math.min( type.max, Math.max( 0, value ) );
```

Comment:
Ah, true. We don't enforce indentation via ESLint yet, it's waiting for https://github.com/eslint/eslint/issues/6690 to get fixed. I applied the change.